t/Controller/Build/evals.t

```diff
@@ -0,0 +1,33 @@
+use strict;
+use Setup;
+use Data::Dumper;
+my %ctx = test_init();
+
+require Hydra::Schema;
+require Hydra::Model::DB;
+require Hydra::Helper::Nix;
+
+use Test2::V0;
+require Catalyst::Test;
+use HTTP::Request::Common;
+Catalyst::Test->import('Hydra');
+
+my $db = Hydra::Model::DB->new;
+hydra_setup($db);
+
+my $project = $db->resultset('Projects')->create({name => "tests", displayname => "", owner => "root"});
+
+my $jobset = createBaseJobset("basic", "basic.nix", $ctx{jobsdir});
+
+ok(evalSucceeds($jobset),               "Evaluating jobs/basic.nix should exit with return code 0");
+is(nrQueuedBuildsForJobset($jobset), 3, "Evaluating jobs/basic.nix should result in 3 builds");
+my ($build, @builds) = queuedBuildsForJobset($jobset);
+
+ok(runBuild($build), "Build '".$build->job."' from jobs/basic.nix should exit with code 0");
```

Comment:
Nit: you use `code 0` here, but `return code 0` above -- unify usage.

```suggestion
ok(runBuild($build), "Build '".$build->job."' from jobs/basic.nix should exit with return code 0");
```

Reply:
:+1: I fixed that up across the test base in a followup commit.